quiz.py

```diff
@@ -3,4 +3,18 @@
 Author: Name
 Date: YYYY-MM-DD
 Description: A brief explanation of what this program does.
-"""
\ No newline at end of file
+"""
+print ("Multiple-Choice Quiz Game")
+print ("")
+print ("1. Which of these agents role is a Duelist?")
+iso = ("(a) Iso")
+kj = ("(b) Killjoy")
+omen = ("(c) Omen")
+print (iso)
+print (kj)
+print (omen)
+a = input("> ")
+if a:
+    print("Correct!")
+else:
+    print("Incorrect.")
```

Comment:
Good use of selection, but your comparison (e.g. `if some_condition`) is not going to work. In fact, you have no comparison at all. Look at [my notes](https://github.com/STE-Technology/ICS3U-Unit-2-Control-Flow#comparison-operators) and make sure you are comparing the user's input `a` with something (maybe another string) using a comparison operator like `==` for equality or `!=` for inequality.